apis/duck/register.go

```diff
@@ -18,4 +18,12 @@ package duck
 
 const (
 	GroupName = "duck.knative.dev"
+
+	// AddressableDuckVersionLabel is the label we use to declare
+	// which Addressable duck version type we conform to.
```

Comment:
It's only used with the value `true` (could even be an empty string) so it's not about the version AFAIK. Or did I miss an update?
```suggestion
	// that a type conforms to the Addressable duck type.
```